src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -194,18 +199,11 @@ public Func<EndpointInfo> OtlpServiceHttpEndPointAccessor
 
         _logger = GetLogger();
 
-        // this needs to be explicitly enumerated for each supported language
-        // our language list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
-        var supportedLanguages = new[]
-        {
-            "en", "cs", "de", "es", "fr", "it", "ja", "ko", "pl", "pt-BR", "ru", "tr", "zh-Hans", "zh-Hant", // Standard cultures for compliance.
-            "zh-CN", // Non-standard culture but it is the default in many Chinese browsers. Adding zh-CN allows OS culture customization to flow through the dashboard.
-            "en-GB", // Support UK DateTime formatting (24-hour clock, dd/MM/yyyy)
-        };
+        var supportedCultures = GetSupportedCultures();
 
         _app.UseRequestLocalization(new RequestLocalizationOptions()
-            .AddSupportedCultures(supportedLanguages)
-            .AddSupportedUICultures(supportedLanguages));
+            .AddSupportedCultures([.. supportedCultures])
+            .AddSupportedUICultures([.. supportedCultures]));
```

Comment:
This creates three collections for the supported cultures, as the `[.. list]` operator here is creating a new array for each.

Perhaps instead `GetSupportedCultures()` should return `string[]` to avoid these two collection copies. It'd simplify the code a little, and allocate slightly less.